apps/list.c

```diff
@@ -1258,6 +1325,8 @@ const OPTIONS list_options[] = {
      "List of public key algorithms"},
     {"public-key-methods", OPT_PK_METHOD, '-',
      "List of public key methods"},
+    {"provider-info", OPT_PROVIDER_INFO, '-',
+     "List of provider information"},
```

Comment:
What about `-providers`? We do list info with other stuff, but their respective options aren't called things like `-signature-algorithm-info`

Reply:
I considered this.  I went this way because we have a `-provider` option, I was attempting to avoid confusion by not calling it `-providers`.

I'm happy to change.

Reply:
I've switched to `-providers` for consistency.

Reply:
Ah, seeing the upstream comment.  We also have ```-digest``` flag and ```-digests```